bcrs-ui/public/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title>hello-world</title>
+    <link rel="icon" href="dummy-not-used-do-not-remove">
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <link rel="preload" href="<%= BASE_URL %>fonts/materialdesignicons-webfont.927457ed.woff2" as="font" type="font/woff2" crossorigin>
```

Comment:
Thanks for including the fonts.
Do a `npm run build` and check the dist folder to confirm this font name.

Reply:
K, good idea, will do!